sewer/cli.py

```diff
@@ -57,7 +57,7 @@ def main():
         "--dns",
         type=str,
         required=True,
-        choices=['cloudflare', 'aurora', 'acmedns', "aliyun", "alicloud"],
+        choices=['cloudflare', 'aurora', 'acmedns', "aliyun", "alicloud", "he", "hedns"],
```

Comment:
@menduo we should use just one `he` or `hedns` 
since we already have `cloudflare`, `aurora` I think we should just call it `hurricane`